utilities/run-test-suites/gsheetsImport/resultsImporter.py

```diff
@@ -31,6 +32,26 @@
                "Not Applicable"]
 
 
+def get_range(worksheet_data, start_row, start_col, end_row, end_col):
```

Comment:
`get_range` is definitely better than the string hackery and `gspread.utils.range` calls we had before